### PR TITLE
html-collection 

### PR DESCRIPTION
**Pull request by OlVin69**

### CODE REVIEW

File: src/partials/collection.html [hunk 1]
```diff
@@ -1,3 +1,53 @@
 <section class="collection">
```
Comment (SergGrishkov) [inline src/partials/collection.html]: 1. Should be add id in section `<section class="collection" id="collection">`

2. After tag section  should be `<div class="collection-container container">`. In container should be added all your code.

File: src/partials/collection.html [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
-
+    <h2 class="collection-title">Our collection</h2>
+    <ul class="collection-list">
+        <li class="collection-item">
+            <img srcset="../img/collection/poltava@2x.jpg 2x" src="../img/collection/poltava.jpg"
```
Comment (SergGrishkov) [inline src/partials/collection.html]: Change path with 1 dot in for all paths below.
exmpl: "../img/collection/poltava@2x.jpg 2x"  => "./img/collection/poltava@2x.jpg 2x" for general standard

File: src/partials/collection.html [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+                alt="august poltava women's blousering" class="collection-item-img">
+            <div class="collection-item-box">
+                <h3 class="collection-item-title">august poltava women's blousering</h3>
+                <p class="collection-item-price">3300 uah</p>
+            </div>
+        </li>
+        <li class="collection-item">
+            <img srcset="../img/collection/gorgany@2x.jpg 2x" src="../img/collection/gorgany.jpg"
```
Comment (SergGrishkov) [inline src/partials/collection.html]: Change path with 1 dot

File: src/partials/collection.html [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+                alt="women's blouse gorgany" class="collection-item-img">
+            <div class="collection-item-box">
+                <h3 class="collection-item-title">women's blouse gorgany</h3>
+                <p class="collection-item-price">3350 uah</p>
+            </div>
+        </li>
+        <li class="collection-item">
+            <img srcset="../img/collection/mountain@2x.jpg 2x" src="../img/collection/mountain.jpg"
```
Comment (SergGrishkov) [inline src/partials/collection.html]: Change path with 1 dot

File: src/partials/collection.html [hunk 5]
```diff
@@ -1,3 +1,53 @@
...
+                alt="mountain men's shirt" class="collection-item-box">
+            <div class="collection-item-box">
+                <h3 class="collection-item-title">mountain men's shirt</h3>
+                <p class="collection-item-price">3300 uah</p>
+            </div>
+        </li>
+        <li class="collection-item">
+            <img srcset="../img/collection/yasen@2x.jpg 2x" src="../img/collection/yasen.jpg" alt="men's shirt yasen"
```
Comment (SergGrishkov) [inline src/partials/collection.html]: Change path with 1 dot

File: src/partials/collection.html [hunk 6]
```diff
@@ -1,3 +1,53 @@
...
+                class="collection-item-img">
+            <div class="collection-item-box">
+                <h3 class="collection-item-title">men's shirt yasen</h3>
+                <p class="collection-item-price">1700 uah</p>
+            </div>
+        </li>
+        <li class="collection-item">
+            <img srcset="../img/collection/crimea@2x.jpg 2x" src="../img/collection/crimea.jpg"
```
Comment (SergGrishkov) [inline src/partials/collection.html]: Change path with 1 dot

File: src/partials/collection.html [hunk 7]
```diff
@@ -1,3 +1,53 @@
...
+                alt="women's blouse crimea" class="collection-item-img">
+            <div class="collection-item-box">
+                <h3 class="collection-item-title">women's blouse crimea</h3>
+                <p class="collection-item-price">4000 uah</p>
+            </div>
+        </li>
+        <li class="collection-item">
+            <img srcset="../img/collection/verhovina@2x.jpg 2x" src="../img/collection/verhovina.jpg"
```
Comment (SergGrishkov) [inline src/partials/collection.html]: Change path with 1 dot

**Review by SergGrishkov — CHANGES_REQUESTED** [review]

File: src/partials/collection.html (hunk 1 above)
Comment (OlVin69) [reply src/partials/collection.html]: Done

File: src/partials/collection.html (hunk 2 above)
Comment (OlVin69) [reply src/partials/collection.html]: Done

File: src/partials/collection.html (hunk 3 above)
Comment (OlVin69) [reply src/partials/collection.html]: done

File: src/partials/collection.html (hunk 4 above)
Comment (OlVin69) [reply src/partials/collection.html]: Done

File: src/partials/collection.html (hunk 5 above)
Comment (OlVin69) [reply src/partials/collection.html]: Done

File: src/partials/collection.html (hunk 6 above)
Comment (OlVin69) [reply src/partials/collection.html]: Done

File: src/partials/collection.html (hunk 7 above)
Comment (OlVin69) [reply src/partials/collection.html]: Done

**Review by SergGrishkov — APPROVED** [review]

**Pull request merged**